### PR TITLE
capture pcaps on the p2p link and make these available to the host

### PR DESCRIPTION
**Pull request by marten-seemann**
DO NOT MERGE. This is based on #24. Will rebase before the merge.

### CODE REVIEW

File: sim/scenarios/helper/quic-point-to-point-helper.cc [hunk 1]
```diff
@@ -14,6 +14,9 @@ void QuicPointToPointHelper::SetQueueSize(StringValue size) {
 
 NetDeviceContainer QuicPointToPointHelper::Install(Ptr<Node> a, Ptr<Node> b) {
   NetDeviceContainer devices = PointToPointHelper::Install(a, b);
+  // capture a pcap of all packets
+  EnablePcap("/logs/trace_node_left.pcap", devices.Get(0), false, true);
+  EnablePcap("/logs/trace_node_right.pcap", devices.Get(1), false, true);
```
Comment (janaiyengar) [inline sim/scenarios/helper/quic-point-to-point-helper.cc]: This is on the server side, right?

**Review by janaiyengar — APPROVED** [review]
LGTM. I've merged #24, so go ahead with this.

File: sim/scenarios/helper/quic-point-to-point-helper.cc (hunk 1 above)
Comment (marten-seemann) [reply sim/scenarios/helper/quic-point-to-point-helper.cc]: This is on both sides of the p2p link.

Comment (janaiyengar) [reply sim/scenarios/helper/quic-point-to-point-helper.cc]: Cool, thanks! I can't keep these device IDs straight.

Comment (marten-seemann) [reply sim/scenarios/helper/quic-point-to-point-helper.cc]: We might want to rename the log files. Let's get this merged and move on from there.

**Review by matabogekego-collab — APPROVED** [review]

**Pull request merged**